vignettes/create_new_project.Rmd

```diff
@@ -0,0 +1,215 @@
+
+See this [guide on testing](https://r-pkgs.org/testing-basics.html) by Hadley Wickham for getting started.
+
+### What is a GitHub .gitignore file?
+
+A .gitignore file is a text file that tells Git which files or directories to ignore in a project. It helps you avoid committing sensitive information, temporary files, or other unnecessary files to your Git repository. By creating a .gitignore file, you can keep your repository clean and focused on the essential files needed for your project. Find more guidance on how to avoid revealing sensitive information on the [Analysts' Guide](https://dfe-analytical-services.github.io/analysts-guide/RAP/rap-statistics.html#avoid-revealing-sensitive-information).
+
+## Using the create_project() function {#using-the-create_project-function}
+
+The `create_project()` function is a part of the `dfeR` package and is used to create a new R project with a custom folder structure. It is designed to help you set up your project quickly and efficiently, with all the necessary components in place.
+
+The function works by:
+
+1.  **Setting up the core project structure**: The core project structure will either create an ad-hoc project folder structure, which includes folders for data, analysis and output, or a larger folder structure for publication projects, which includes more folders for different data, analysis, documentation and outputs.
+
+2.  **Adding initial files**: The script creates some initial files, such as a template README file to help you get started with your project. It also initialises the `testthat` package, which is used for unit testing in R.
+
+3.  **Customisation**: Other files or processes can also be initialised depending on whether you set extra arguments as `TRUE` or `FALSE`. For example, you can use customisation to create a GitHub .gitignore file or initialise the `renv` package.
+
+### Where to save your project
+
+You should store any projects that use Git outside of OneDrive. To create a folder in your C drive, type: 'C:/Users/username' into the File Explorer search bar and create a folder called 'repos'.
+
+### Parameters
+
+When creating a project using the `create_project()` function, you will need to decide on several parameters. These parameters are necessary whether you use code or the 'New Project Wizard' to create your project. If you are using the 'New Project Wizard', the parameters which need a `TRUE`/`FALSE` input will be set to `TRUE` by ticking a checkbox. The table below explains each of the parameters, their meaning and what you should take into account for each.
+
+```{r, echo = FALSE, warning=FALSE, message=FALSE}
```

Comment:
Neat use of kable / R to make the table!

![image](https://github.com/user-attachments/assets/c1404b4a-2fdf-4a36-95d1-d87bdb65ca03)